zone/zonedb.cpp

```diff
@@ -1378,6 +1380,7 @@ bool ZoneDatabase::SaveCharacterData(uint32 character_id, uint32 account_id, Pla
 		Strings::Escape(pp->title).c_str(),						  // " title,                     "
 		Strings::Escape(pp->suffix).c_str(),						  // " suffix,                    "
 		pp->exp,						  // " exp,                       "
+		m_epp->exp_enabled,				  // " exp_enabled,               "
```

Comment:
We won't have access to the client object in this method so lets just pass the client pointer for now and pull things out of the client object within this save routine.

I'd avoid converting this to repositories for now

All of this client save stuff needs to get cleaned up

Reply:
Yeah the reason I put it in extended profile is that we didn’t pass anything but player profile and extended profile to this method.